composer-require-checker.json

```diff
@@ -1,3 +1,9 @@
 {
-  "scan-files" : ["bin/esb"]
+  "scan-files" : ["bin/esb"],
+  "symbol-whitelist": [
+    "null", "true", "false", "static", "self", "parent",
+    "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object", "mixed", "never",
```

Comment:
Why are those core symbols whitelisted? There weren't the need to whitelist them previously...